xarm_gazebo/launch/_robot_beside_table_gazebo.launch.py

```diff
@@ -53,32 +42,30 @@ def build_robot_description(
             }
         ),
     }
-
     return robot_description
 
-def build_camera_description(this_robot_namespace=""):
-    robot_description = {
+def build_camera_description(camera_namespace=""):
```

Comment:
@NikolaRaicevic2001 Let's separate this to another launch file in the future.

Reply:
Yes, that will be the part of camera_launch file